### ISSUE TITLE
Using spotweb behind a vpn

### ISSUE THREAD

**Issue by betonishard** [jgeusebroek/docker-spotweb#19]
Hi all,

I have been looking for an answer and tried so many things already so I thought to ask it here. It is not a bug but a request for information.

I have a vpn container to nordpvn (bubuntux). I would like to connect spotweb via that VPN to the outside world. So first I changed the ports. SO defaulting to 8080 in stead of port 8080 by changing 
- 000-default.conf 
- ports.conf
and mount these files. This works when I run it without VPN.

yaml example: 
spotweb:
    image: jgeusebroek/spotweb
    user: '0'
    container_name: spotweb
    network_mode: host
    env_file:
      - spotweb.env
    volumes:
      - ~/container/apps/spotweb/data:/config
      - ~/container/apps/spotweb/data/ports.conf:/etc/apache2/ports.conf
      - ~/container/apps/spotweb/data/000-default.conf:/etc/apache2/sites-enabled/000-default.conf
    ports:
      - 8080:8080
(commented)   network_mode: service:vpn
    restart: unless-stopped

However when I uncomment network mode:service:vpn,comment the ports and network mode: host, it is not reachable. VPN works with other services like SONARR / RADARR etc, and I can go to the url...

To sum up:
Separate VPN works
Other services via the same VPN container also work (e.g. sonarr 10.1.0.100:8989 is reachable)
Separate spotweb works (e.g. 10.1.0.100:8080)
Combine Spotweb via that VPN does not work (e.g. 10.1.0.100:8080)

Does someone have experience with this and point me in the correct direction:

Thank you for your help it would be much appreciated...

**Comment by betonishard** [comment]
Found the issue... finally...
